command.go

```diff
@@ -341,6 +341,10 @@ func (c *Command) Find(arrs []string) (*Command, []string, error) {
 		return c.Root(), arrs, nil
 	}
 
+	for len(arrs) > 0 && len(arrs[0]) == 0 {
```

Comment:
It related to the fact that root commands have special code to ignore it later. So I'm just ripping them out for all commands.